react/features/base/dialog/components/Dialog.web.js

```diff
@@ -31,6 +27,11 @@ class Dialog extends AbstractDialog {
         isModal: React.PropTypes.bool,
 
         /**
+         * Disables rendering of the submit button.
+         */
+        submitDisabled: React.PropTypes.bool,
+
```

Comment:
Yep, I'm wrong. But anyway I think we can rename submitDisabled, cause we have okDisabled and this is confusing ... but this is not to be fixed in this PR.